common/src/main/java/com/movie/recommender/common/model/MovieShowtimesResponse.java

```diff
@@ -0,0 +1,66 @@
+package com.movie.recommender.common.model;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class MovieShowtimesResponse {
```

Comment:
split this class into:

- MovieShowtimesResponse
- SearchMetadata
- SearchParameters
- SearchInformation
- Showtime (includes Showtime, Theater, and Showing)

only Showtime should be in the common project
move the other classes to the crawler project (create a model package there and place these classes)